src/components/BackComponent.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button, Col, Row } from "reactstrap";
+
+const BackComponent = () => {
```

Comment:
Why do you need to create this component if you just use this in one place? @azukhrufy